h/static/scripts/app.coffee

```diff
@@ -93,50 +76,81 @@ setupStreamer = [
-
-require('./annotation-sync')
-require('./annotation-ui-sync')
-require('./bridge')
-require('./discovery')
+.controller('AppController', require('./app-controller'))
+.controller('AnnotationController', require('./annotation-controller'))
+.controller('AnnotationUIController', require('./annotation-ui-controller'))
+.controller('StreamController', require('./stream-controller'))
+.controller('WidgetController', require('./widget-controller'))
+
+.directive('annotation', require('./directive/annotation'))
+.directive('deepCount', require('./directive/deep-count'))
+.directive('formInput', require('./directive/form-input'))
+.directive('formValidate', require('./directive/form-validate'))
+.directive('markdown', require('./directive/markdown'))
+.directive('privacy', require('./directive/privacy'))
+.directive('repeatAnim', require('./directive/repeat-anim'))
+.directive('simpleSearch', require('./directive/simple-search'))
+.directive('statusButton', require('./directive/status-button'))
+.directive('thread', require('./directive/thread'))
+.directive('threadFilter', require('./directive/thread-filter'))
+.directive('whenscrolled', require('./directive/whenscrolled'))
+.directive('match', require('./directive/match'))
+.directive('tabbable', require('./directive/tabbable'))
+.directive('tabReveal', require('./directive/tab-reveal'))
+
+.filter('converter', require('./filter/converter'))
+.filter('moment', require('./filter/moment'))
+.filter('persona', require('./filter/persona'))
+.filter('urlencode', require('./filter/urlencode'))
+
+.provider('identity', require('./identity'))
+.provider('session', require('./session'))
+
+.service('annotator', -> new Annotator(angular.element('<div>')))
+.service('annotationMapper', require('./annotation-mapper'))
+.service('annotationUI', require('./annotation-ui'))
+.service('auth', require('./auth'))
+.service('bridge', require('./bridge'))
+.service('crossframe', require('./cross-frame'))
+.service('drafts', require('./drafts'))
+.service('flash', require('./flash'))
+.service('formRespond', require('./form-respond'))
+.service('host', require('./host'))
+.service('localStorage', require('./local-storage'))
+.service('permissions', require('./permissions'))
+.service('pulse', require('./pulse'))
+.service('queryParser', require('./query-parser'))
+.service('render', require('./render'))
+.service('searchFilter', require('./search-filter'))
+.service('store', require('./store'))
+.service('streamFilter', require('./stream-filter'))
+.service('streamer', require('./streamer'))
+.service('tags', require('./tags'))
+.service('time', require('./time'))
+.service('threading', require('./threading'))
+.service('unicode', require('./unicode'))
+.service('viewFilter', require('./view-filter'))
+
+.value('xsrf', token: null)
+.value('AnnotationSync', require('./annotation-sync'))
+.value('AnnotationUISync', require('./annotation-ui-sync'))
+.value('Discovery', require('./discovery'))
+
+.run(setupCrossFrame)
```

Comment:
As far as I know. Seems to work!
On Mar 26, 2015 9:55 AM, "Nick Stenning" <notifications@github.com> wrote:

> In h/static/scripts/app.coffee
> <https://github.com/hypothesis/h/pull/2092#discussion_r27232152>:
>
> > +.service('searchFilter', require('./search-filter'))
> > +.service('store', require('./store'))
> > +.service('streamFilter', require('./stream-filter'))
> > +.service('streamer', require('./streamer'))
> > +.service('tags', require('./tags'))
> > +.service('time', require('./time'))
> > +.service('threading', require('./threading'))
> > +.service('unicode', require('./unicode'))
> > +.service('viewFilter', require('./view-filter'))
> > +
> > +.value('xsrf', token: null)
> > +.value('AnnotationSync', require('./annotation-sync'))
> > +.value('AnnotationUISync', require('./annotation-ui-sync'))
> > +.value('Discovery', require('./discovery'))
> > +
> > +.run(setupCrossFrame)
>
> Just double checking: this is no longer conditional on being in the test
> environment because the tests no longer load the tested services through
> this angular module.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hypothesis/h/pull/2092/files#r27232152>.
>